.github/workflows/release.yml

```diff
@@ -26,81 +26,80 @@ jobs:
     name: "Perform any build or bundling steps, as necessary."
     uses: ./.github/workflows/build.yml
 
-  # ssdlc:
-  #   needs: [release_please, build]
-  #   permissions:
-  #     # required for all workflows
-  #     security-events: write
-  #     id-token: write
-  #     contents: write
-  #   environment: release
-  #   runs-on: ubuntu-latest
-  #   steps:
-  #     - uses: actions/checkout@v4
+  ssdlc:
+    needs: [release_please, build]
+    permissions:
+      # required for all workflows
+      security-events: write
+      id-token: write
+      contents: write
+    environment: release
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
 
-  #     - name: Install Node and dependencies
-  #       uses: mongodb-labs/drivers-github-tools/node/setup@v2
-  #       with:
-  #         ignore_install_scripts: true
```

Comment:
why do we ignore the install scripts here?

Reply:
We do the same for kerberos and mongodb-client-encryption.

By default, `npm i` on our packages with native bindings downloads a prebuilt copy of our C++ code (there is an `install` script) or attempts to compile the bindings.  All we need for this step is the JS dependencies